src/components/Navigation/TopBar/Help/ReleaseNotesDialog/ReleasePosts/ReleasePost.tsx

```diff
@@ -0,0 +1,119 @@
+import { FC, useState } from 'react';
+
+import { Icon, Typography } from '@equinor/eds-core-react';
+import { arrow_drop_down, arrow_drop_up } from '@equinor/eds-icons';
+
+import ReleaseNotesTypes from '../ReleaseNotesTypes/ReleaseNotesTypes';
+import { ReleaseNoteType } from '../ReleaseNotesTypes/ReleaseNotesTypes.types';
+import TextContent from '../TextContent';
+import {
+  AccordionText,
+  BodyContainer,
+  BtnContainer,
+  Button,
+  Container,
+  HeadingContainer,
+  ReleaseNoteTypeContainer,
+  RightContainer,
+  TitleContainer,
+  TopContainer,
+} from './ReleasePost.styles';
+import { ReleaseNote } from 'src/api/models/ReleaseNote';
+import { date } from 'src/utils';
+
+import { AnimatePresence, motion } from 'framer-motion';
+
+const ReleasePost: FC<ReleaseNote> = ({
+  createdDate,
+  version,
+  tags,
+  title,
+  body,
+}) => {
+  const [showFullContent, setShowFullContent] = useState<boolean>(false);
+  const [needsShowMoreButton, setNeedsShowMoreButton] = useState(false);
+
+  // TODO: Could not get the test to work properly (get the height from the element), created a task for it (BUG 438384 in Azure board)
+  /* c8 ignore start */
+  const handleOnNewRef = (element: HTMLDivElement | null) => {
+    if (element) {
+      const { clientHeight } = element;
+      setNeedsShowMoreButton(clientHeight > 108);
+    }
+  };
+  /* c8 ignore end */
+
+  const toggleContent = () => {
+    setShowFullContent((prev) => !prev);
+  };
+  const buttonText = showFullContent ? 'Read less' : 'Read more';
+
+  return (
+    <Container>
+      <AnimatePresence>
```

Comment:
AnimatePresence isn't needed here, it's only used when the element inside it is supposed to be removed from the DOM

Reply:
Removed